src/containers/AuthorPageContainer/useAuthorPageContainer.tsx

```diff
@@ -18,12 +18,40 @@ export const useAuthorPageContainer = (): AuthorPageProps | undefined => {
     return;
   }
 
+  const { author, posts } = resultAuthorPage;
   return {
     backLink: <BackLinkContainer />,
-    author: resultAuthorPage.author,
+    author: {
+      username: author.username,
+      name: author.name,
+      avatarImageUrl: author.avatarImageUrl,
+      socialNetworks: Object.entries(author.socialNetworks || {}).map(([name, username]) => {
+        const socialNetworkName = name as SocialNetworkName;
```

Comment:
```suggestion
        const socialNetworkName = name;
```

When doing this https://github.com/eleven-labs/blog.eleven-labs.com/pull/953/files#r1238254160 you will not need to convert it into `SocialNetworkName` type

Reply:
With Object.entries the keys become strings and we lose the SocialNetworkName typing